db/migrate/20211202105727_create_posts.rb

```diff
@@ -3,8 +3,8 @@ def change
     create_table :posts do |t|
       t.string :title
       t.text :text
-      t.integer :commentsCounter
-      t.integer :likesCounter
+      t.integer :comments_counter
+      t.integer :likes_counter
 
       t.timestamps
     end
```

Comment:
- [ ] Post model should have a method that returns 3 most recent posts
- [ ] Post model should have a method that returns the 5 most recent comments for a given post
- [ ] Comment model should have a method that updates the comments counter for a post
- [ ] Like model should have a method that updates the likes counter for a post.
- [ ] ApplicationController should contains `#current_user` method